constraint/pkg/apis/addtoscheme_templates_v1alpha1.go

```diff
@@ -15,11 +15,7 @@ limitations under the License.
 
 package apis
 
-import (
-	"github.com/open-policy-agent/frameworks/constraint/pkg/apis/templates/v1alpha1"
```

Comment:
dropping the import seems suspicious to me. If we aren't importing v1beta1, how are we adding its scheme?